src/jit/codegenxarch.cpp

```diff
@@ -1367,6 +1367,15 @@ void CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
             {
                 assert(!isRegCandidate);
 
+#if defined(FEATURE_SIMD) && defined(_TARGET_X86_)
+                // Loading of TYP_SIMD12 (i.e. Vector3) variable
+                if (treeNode->TypeGet() == TYP_SIMD12)
+                {
+                    genLoadLclTypeSIMD12(treeNode);
+                    break;
+                }
+#endif // defined(FEATURE_SIMD) && defined(_TARGET_X86_)
```

Comment:
What about spill and reload of TYP_SIMD12 locals?  Don't we need to modify genProduceReg() and genConsumeReg()/genUnspillRegIfNeeded() code paths to correctly handle TYP_SIMD12 locals (or rather args)?